diff -Nur jftpgw-0.13.1.orig/Makefile.in jftpgw-0.13.1/Makefile.in
--- jftpgw-0.13.1.orig/Makefile.in	Mon Nov 25 23:11:11 2002
+++ jftpgw-0.13.1/Makefile.in	Mon Nov 25 23:12:12 2002
@@ -529,9 +529,9 @@
 # don't install a list of file. Just one file per install.
 # if you have more of them, create a subdirectory with an extra Makefile
 install-data-local:
-	$(mkinstalldirs) $(sysconfdir)
-	if test -f $(sysconfdir)/jftpgw.conf; then echo -e "\n\nMoving $(sysconfdir)/jftpgw.conf   to   $(sysconfdir)/jftpgw.conf.old\n\n"; mv $(sysconfdir)/jftpgw.conf $(sysconfdir)/jftpgw.conf.old; fi
-	$(INSTALL_DATA) jftpgw.conf.sample $(sysconfdir)/jftpgw.conf
+	$(mkinstalldirs) $(DESTDIR)$(sysconfdir)
+	if test -f $(DESTDIR)$(sysconfdir)/jftpgw.conf; then echo -e "\n\nMoving $(DESTDIR)$(sysconfdir)/jftpgw.conf   to   $(DESTDIR)$(sysconfdir)/jftpgw.conf.old\n\n"; mv $(DESTDIR)$(sysconfdir)/jftpgw.conf $(DESTDIR)$(sysconfdir)/jftpgw.conf.old; fi
+	$(INSTALL_DATA) jftpgw.conf.sample $(DESTDIR)$(sysconfdir)/jftpgw.conf
 
 # remove ALL you have installed in install-data-local or install-exec-local
 uninstall-local:
